### PR TITLE
Add earnings page

### PR DESCRIPTION
**Pull request by renzo-x**
## Description
This PR adds an earning page for the payroll system.

image

## Type of change
- New feature (adds new functionality)

### CODE REVIEW

**Comment by torodeveloper** [comment]
Do you have screenshots of how the pages work?

**Comment by torodeveloper** [comment]
This PR also has conflicts with the latest master branch. Please fix it.

File: components/delete/index.json
Comment (torodeveloper) [inline components/delete/index.json:1]: Rename it to `delete-earning` please

Comment (renzo-x) [reply components/delete/index.json:1]: Done sir.

**Review by torodeveloper — APPROVED** [review]
LGTM!

**Pull request merged**